examples/stable-diffusion/text_to_image_generation.py

```diff
@@ -441,7 +441,7 @@ def main():
     kwargs_call["quant_mode"] = args.quant_mode
 
     # Instantiate a Stable Diffusion pipeline class
-    import habana_frameworks.torch.core as htcore  # noqa: F401
+    #import habana_frameworks.torch.core as htcore  # noqa: F401
```

Comment:
If we don't need this, we should rather remove it

Reply:
@regisss @skaulintel  We need this for quantization.  If we remove we are getting weird error.   I will investigate if there is some other workaround that can work instead. Notice that I had to add ` # noqa: F401` because ruff will else remove it whe we do `make style`..

Reply:
It's fine to have `# noqa:`. I don't think it's an issue to keep the import here, or have commented it because it doesn't work @skaulintel ?

Reply:
We kind of have "catch-22 problem" here I think:
1. If we remove, we will get error with quantization but will resolve issue in this PR
2. If we keep, quantization works, but this PR has issue
..

Reply:
I will remove this change for now then. @dsocek Could you open a separate PR to address this when you find a fix?

Reply:
@skaulintel  sounds good.  I will work on this PR